test/linux.rs

```diff
@@ -0,0 +1,71 @@
+cfg_if! {
+    if #[cfg(target_os = "linux")] {
+        mod linux_tests {
+            use super::super::*;
+
+            const MOCK_DIR: &str = "linux/data/zfs";
+
+            fn tank_set() -> HashSet<Snapshot> {
+                [
+                    Snapshot {
+                        name: "tank/vm/chimera".to_string(),
+                        writes: 2391634,
+                        nwritten: 3953804,
+                        reads: 55453,
+                        nread: 2032404,
+                        nunlinks: 451747,
+                        nunlinked: 1441696,
```

Comment:
I used file order for these, because I copied the text from my local files and used macros in my editor to convert it to rust source code.  I was too lazy to change to struct definition order and I assume it is not an issue.